testprogs/selpolltest.c

```diff
@@ -74,7 +74,7 @@ main(int argc, char **argv)
 	struct bpf_program fcode;
 	char ebuf[PCAP_ERRBUF_SIZE];
 	pcap_if_t *devlist;
-	int selectable_fd;
```

Comment:
This change is already in the main branch from commit fbb5e9eec8bac69c7eb21e97ed5b20751f54290e (and is unrelated to the I/O plugins).